src/app/shared/common/confirmation-dialog/confirmation-dialog.component.ts

```diff
@@ -64,16 +65,23 @@ export class ConfirmDialogModel {
   }
 
   public static forDelete(
-    type: string,
+    typeKey: string,
     identifier: string,
+    translateService: TranslateService,
   ): ConfirmDialogModel {
     const dialogData = new ConfirmDialogModel(
-      'Deletion confirmation',
-      `Please confirm you want to delete ${type} ${identifier}. This action cannot be undone.`,
+      `${translateService.instant('component_library.delete_title')}`,
+      `${translateService.instant(
+        'component_library.delete_one',
+      )} ${translateService.instant(
+        typeKey,
+      )} ${identifier}. ${translateService.instant(
+        'component_library.delete_two',
+      )}`,
     );
     dialogData.confirmColor = 'warn';
-    dialogData.confirmText = 'Delete';
-    dialogData.cancelText = 'Cancel';
+    dialogData.confirmText = translateService.instant('general.delete');
+    dialogData.cancelText = translateService.instant('general.close');
```

Comment:
Is this a deliberate choice to go from "Cancel" to "Close"? If not, it's the wrong key. It should be `general.cancel`.

Reply:
Yes, it's an abstract confirmation dialog, that could be "cancel" or "close". In case of a deletion you want it to be "[red] Delete | Close"